src/model/globalDefault.js

```diff
@@ -67,15 +67,15 @@ export default {
 
     animationThreshold: 2000,
     // Configuration for progressive/incremental rendering
-    progressiveThreshold: 3000,
+    progressiveThreshold: Infinity,
     progressive: 400,
 
     // Threshold of if use single hover layer to optimize.
     // It is recommended that `hoverLayerThreshold` is equivalent to or less than
     // `progressiveThreshold`, otherwise hover will cause restart of progressive,
     // which is unexpected.
     // see example <echarts/test/heatmap-large.html>.
-    hoverLayerThreshold: 3000,
+    hoverLayerThreshold: Infinity,
```

Comment:
I think The default hoverLayerThreshold should not be simply set Infinity.
I think we should find a better solution in `updateHoverLayerStatus`